validators.py

```diff
@@ -1,7 +1,23 @@
-from pydantic import BaseModel, conlist, conint, Field
+from datetime import datetime
+
+from pydantic import BaseModel, validator
 
 
 class DatesAndPriceCheckInputValidator(BaseModel):
-    listing_ids: conlist(conint(gt=0), min_items=1) = Field(
-        title="Listing ids to check on"
-    )
+    listing_ids: list
+    desired_checkin: str
+
+    @validator('listing_ids')
+    def listing_must_greater_than_zero(cls, v):
+        if len(v) < 0:
```

Comment:
Should be `<= 0` because an empty list should throw an error

Reply:
Noted. Will make the change